### PR TITLE
fixes issue #147 add an option in setting to disable STT

### PR DESCRIPTION
**Pull request by amitiwary999**
fixes issue #147 add an option in setting to disable STT.

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -96,7 +96,7 @@
     private int offset = 1;
     private ChatFeedRecyclerAdapter recyclerAdapter;
     private Realm realm;
-
+    Boolean mici;
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Please use better variable names.

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 2]
```diff
@@ -310,6 +310,35 @@ public void setChatBackground() {
...
+            });
+        }
+        else {
+            btnSpeak.setImageResource(R.drawable.ic_send_fab);
+            btnSpeak.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    switch (v.getId()) {
+                        case R.id.btnSpeak:
+                            String message = etMessage.getText().toString();
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Same goes for here as well, etMessage makes no sense, so try to rename it to reflect its purpose in a better way.

**Comment by the-dagger** [comment]
@amitiwary999 Please have a look at the review notes.

**Comment by amitiwary999** [comment]
@the-dagger etMessage is declared in MainActivity by someone.(should I rename it?)

**Comment by the-dagger** [comment]
Yes, please do.
Also try find all instances of incorrectly named variables and fix them in your upcoming Pull Requests. :)

**Pull request closed without merging**